README.md

```diff
@@ -68,78 +68,105 @@ In the `Artifacts` directory, you can find the compiled static library and heade
 
 # Using the SDK
 
-1. Once you know the email *(Preferred)* or userId of the user, **create a shared instance of an `IterableAPI`**
-  * EMAIL: `[IterableAPI sharedInstanceWithApiKey:(NSString *)apiKey andEmail:(NSString *)email launchOptions:(NSDictionary *)launchOptions]` to create an `IterableAPI`
-  * USERID: `[IterableAPI sharedInstanceWithApiKey:(NSString *)apiKey andUserId:(NSString *)userId launchOptions:(NSDictionary *)launchOptions]` to create an `IterableAPI`
-	  * If you are initializing using a userId, an existing user must already exist for that userId 
-	  * It is preferred that you initialize by Email since that doesn't require an additional lookup by userId call on the backend.
+1. On application launch (`application:didFinishLaunchingWithOptions:`), initialize the Iterable SDK:
+
+```objective-c
+IterableConfig *config = [[IterableConfig alloc] init];
+config.pushIntegrationName = "myPushIntegration";
+[IterableAPI initializeWithApiKey:@"<your-api-key>" launchOptions:launchOptions config:config];
+```
   * The `apiKey` should correspond to the API key of your project in Iterable. If you'd like, you can specify a different `apiKey` depending on whether you're building in `DEBUG` or `PRODUCTION`, and point the SDK to the relevant Iterable project.
   * Ideally, you will call this from inside `application:didFinishLaunchingWithOptions:` and pass in `launchOptions`. This will let the SDK automatically track a push open for you if the application was launched from a remote Iterable push notification. 
-  * This method creates a singleton `IterableAPI` for your use. You can retrieve it later with `[IterableAPI sharedInstance]`. If retrieving it later, be sure that you have either instantiated it earlier, or check for a non-nil return value. 
+  * This method creates a singleton `IterableAPI` for your use. You can retrieve it later with `[IterableAPI sharedInstance]`. If retrieving it later, be sure that you have either instantiated it earlier, or check for a non-nil return value.
+
+2. Once you know the email *(Preferred)* or userId of the user, call `setEmail:` or `setUserId:`
```

Comment:
We will need to update this documentation of setEmail/setUserID once we release the change for stand alone userIDs.